Add paid column to debts

YarekTenko opened the pull request:
This column will help determine which debt has been paid.

qr8r commented:
As an alternative to a boolean field, this could also be a date field `:payment_date` Which gives you a little more information

LeonTenko committed:
Add payment_date column to debts

This column will help determine which debt has been paid for and when it
happened.
